libbeat/docs/version.asciidoc

```diff
@@ -1,7 +1,9 @@
-:stack-version: 8.0.0
-:doc-branch: master
+:stack-version: 7.6.0
+:doc-branch: 7.x
 :go-version: 1.12.12
 :release-state: unreleased
 :python: 2.7.9
 :docker: 1.12
 :docker-compose: 1.11
+:branch: 7.x
+:ecs_version: 1.1
```

Comment:
Delete this line - we're getting the setting for `ecs_version` from the shared file in the docs repo now.